pkg/parameters/parameterstore.go

```diff
@@ -0,0 +1,86 @@
+package parameters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cnabio/cnab-go/utils/crud"
+)
+
+// TODO: very much a clone of cnab-go/credentials/credstore.go
```

Comment:
You could have something more generalized, it would require also working with interface{} when unmarshaling and accepting/returning values though... I do that in porter's exec/builder package. It's not easy to read though so I don't recommend it. 

Again, it's go, no generics so some classes are very similar with the types different. 🤷‍♀️